src/java.base/share/classes/java/lang/Class.java

```diff
@@ -779,10 +779,9 @@ public boolean isAnnotation() {
     }
 
     /**
-     * Returns {@code true} if and only if this class has the synthetic modifier
-     * bit set.
+     *{@return {@code true} if and only if this class has the synthetic modifier
```

Comment:
The braces around return tag are allowed?  "{@return ...} vs. "@return ..." Seems inconsistent with other uses of return tags.